scripts/modules/elastic_stack.py

```diff
@@ -76,7 +75,8 @@ def __init__(self, **options):
         elif self.at_least_version("7.3"):
             self.apm_server_command_args.extend([
                 ("apm-server.kibana.enabled", "true"),
-                ("apm-server.kibana.host", self.DEFAULT_KIBANA_HOST)])
+                ("apm-server.kibana.host",
+                    "{}".format(self.options.get("apm_server_kibana_url")))])
```

Comment:
Why is this `format()` necessary? Why not just:

`("apm-server.kibana.host", self.options.get("apm_server_kibana_url", ""))`

With the way it is written if the `get()` fails, the string will get set to the string `None`, which seems like it is probably not what we want?

Reply:
I realized now why `("apm-server.kibana.host", self.options.get("apm_server_kibana_url"))` does no pass the tests, because the test that uses render does not apply the default values so the value assigned is None, so yes this is also valid and easy.